src/main/kotlin/com/bugsnag/android/gradle/BugsnagPlugin.kt

```diff
@@ -66,12 +68,22 @@ class BugsnagPlugin : Plugin<Project> {
 
     @Suppress("LongMethod")
     override fun apply(project: Project) {
+        if (AgpVersions.CURRENT >= AgpVersions.VERSION_7_0) {
+            throw StopExecutionException(
+                "Using com.bugsnag.android.gradle with Android Gradle Plugin 7+ " +
+                    "requires an upgrade to com.bugsnag.android.gradle:7.0.0. " +
+                    "For more information about this change, see " +
+                    "https://docs.bugsnag.com/platforms/android/"
```

Comment:
nit: the link will probably end up here instead: https://docs.bugsnag.com/build-integrations/gradle/